targets/csharp/templates/PlayFabUtil.cs.ejs

```diff
@@ -49,6 +51,100 @@ namespace PlayFab
                 return null;
             return error.GenerateErrorReport();
         }
+
+#if NET45 || NETCOREAPP2_0
+        [ThreadStatic]
+        private static StringBuilder _sb;
+        /// <summary>
+        /// A threadsafe way to block and load a text file
+        /// 
+        /// Load a text file, and return the file as text.
+        /// Used for small (usually json) files.
+        /// </summary>
+        public static string ReadAllFileText(string filename)
+        {
+            if (!File.Exists(filename))
```

Comment:
Here and elsewhere, please use braces even for single statement blocks.

Reply:
This was not my code, it's just me reviving code they had and got rid of. I'll fix it.

Reply:
Fixing this shortly.